pam/internal/gdm/gdm.proto

```diff
@@ -106,7 +106,9 @@ enum EventType {
     // EventType_isAuthenticatedRequested is an authentication request EventType.
     isAuthenticatedRequested = 10;
     // EventType_stageChanged is stage changed EventType.
-    stageChanged = 11;
+    stageChanged = 12;
+    // EventType_isAuthenticationCancelled is authentication cancellation EventType.
+    isAuthenticatedCancelled = 11;
```

Comment:
10 -> 12 -> 11, ouch...

Can we keep the numerical order here?

Reply:
Oh, this meant to be before stageChanged, reason why the order wasn't respected.